dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -301,6 +301,9 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 	const showKeyEventsCarousel =
 		CAPIArticle.config.abTests.keyEventsCarouselVariant == 'variant';
 
+	const isInFilteringBeta =
+		CAPIArticle.config.abTests.automaticFiltersVariant == 'variant';
```

Comment:
Maybe this needs testing, but I think we could have scenarios where `topics` is not provided from frontend meaning we don't want to add filter for this blog, but the `automaticFiltersVariant` could still be `variant`

Reply:
Thanks for the suggestion Marjan, this has been implemented now by adding an additional check for topics.